core/src/main/java/pl/agh/edu/screen/ScenariosScreen.java

```diff
@@ -0,0 +1,339 @@
+package pl.agh.edu.screen;
+
+import java.util.ArrayList;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.InputMultiplexer;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.Touchable;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.scenes.scene2d.utils.NinePatchDrawable;
+import com.badlogic.gdx.utils.Align;
+import com.badlogic.gdx.utils.Scaling;
+
+import pl.agh.edu.GdxGame;
+import pl.agh.edu.actor.GameSkin;
+import pl.agh.edu.actor.component.button.DifficultyButton;
+import pl.agh.edu.actor.component.button.ScenarioButton;
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.enums.DifficultyLevel;
+import pl.agh.edu.enums.HotelType;
+
+public class ScenariosScreen implements Screen {
+	private final Stage stage = new Stage(GraphicConfig.getViewport());
+	private final Skin skin = GameSkin.getInstance();
+	private final Stack stack = new Stack();
+
+	private final Table mainTable = new Table();
+	private final Table difficultyTable = new Table();
+	private final Table scenariosTable = new Table();
+	private final GdxGame game;
+
+	NinePatchDrawable buttonBackground = new NinePatchDrawable(skin.getPatch("button"));
+	NinePatchDrawable onSelect = new NinePatchDrawable(skin.getPatch("button_selected"));
+	private Label.LabelStyle titleLabel;
+
+	// for difficulty
+	public ArrayList<DifficultyButton> myButtons = new ArrayList<>();
```

Comment:
nazwa nie oddaje to czego powwina raczej powino być difficultyButtonList albo coś tego rodzaju